miner/worker.go

```diff
@@ -269,7 +265,14 @@ func (self *worker) update() {
 	}
 	defer self.chainHeadSub.Unsubscribe()
 	defer self.chainSideSub.Unsubscribe()
-	timeout := time.NewTimer(waitPeriod * time.Second)
+
+	// timeout waiting for v1 inital value
+	// TODO: minePeriod := self.engine.(*XDPoS.XDPoS).Config.waitPeriod
```

Comment:
Do you need to finish this TODO? If not, I approve this PR

Reply:
update comment, should do this after we finialize where is the config placement.